pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Please do the following to make sure your changes make it all the way out the door:
+
+- [ ] Make sure your PR updates the version number in ``setup.py`` and ``package.json``
+- [ ] Get a green Travis build for this PR
+- [ ] Merge to master
+- [ ] Create a release tag on GitHub https://github.com/edx/edx-ora2/releases
+- [ ] Manually inspect diff at https://github.com/edx/edx-ora2/compare/0.x.n-1...0.x.n, email contributors
+- [ ] Create PR to update `requirements/edx/{github.in,base.txt,development.txt,testing.txt}` in `edx-platform`.
+- [ ] If manual testing of the changes against edx-platform is desired, create a sandbox (or use devstack).
```

Comment:
I think the sandbox testing step is out of sequence. Wouldn't you need to merge platform pr with the version bump in order to have platform know to use the latest version of ORA?

Reply:
You can create a sandbox against a given branch of edx-platform, like the one that contains your new ORA version.